index.html

```diff
@@ -0,0 +1,58 @@
+﻿<!doctype html>
+	<head>
+		<meta charset="utf-8">
+		<title>Cookies</title>
+		<link rel="stylesheet" href="css/style.css">
+	</head>
+	<body>
+
+		<h1>The Very Best Chocolate Chip Cookies</h1>
+
+		<img src="cookies.jpg" width="400">
+
+		<h3>Recipe by: My Grandma</h3>
+
+
+		<h2>Prep Time</h2>
+		<p>45 Min</p>
+		
+		<h2>Ingredients</h2>
+		<ul>
+			<li>1 1/2 cup (3 sticks) butter, softened</li>
```

Comment:
The indention here is off, you should only use 2 to 4 spaces for indentation (preferably two).